### ISSUE TITLE
[Feature] Confirm media deletion on stop watching

### ISSUE THREAD

**Issue by ouattararomuald** [lardbit/nefarious#263]
I was a bit surprised that, after clicking on "stop watching" it deleted the movie from my disk. The feature is good, but I think a popup (or something similar) should be displayed to the user with a checkbox to select whether they also want to delete the media file.

**Comment by lardbit** [comment]
Yes, I certainly agree.  It's what I want the majority of the time, but it is annoying when you want to keep the file on disk, you have to manually remove the torrent from Transmission, then you can click "stop watching".  It's the current workaround.

I guess this would amount to the UI change for the button, and conditionally updating the transmission api call to maintain the files, in `utils.destroy_transmission_result()`

    transmission_client.remove_torrent([instance.transmission_torrent_hash], delete_data=True, timeout=10)
